backend/habragasolina/src/main/java/bo/edu/usfa/gasolina/habragasolina/Controllers/GasStationController.java

```diff
@@ -0,0 +1,45 @@
+package bo.edu.usfa.gasolina.habragasolina.Controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.repository.query.Param;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import bo.edu.usfa.gasolina.habragasolina.Entities.GasStationEntity;
+import bo.edu.usfa.gasolina.habragasolina.Service.GasStationService;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@RestController
+@RequestMapping("/gasstation")
+public class GasStationController {
+
+    private final GasStationService gasStationService;
+
+    @Autowired
+    public GasStationController(GasStationService gasStationService) {
+        this.gasStationService = gasStationService;
+    }
+
+    @PostMapping("/gasstation")
```

Comment:
PostMapping should only refer to the requestMapping

Reply:
No need to put the "/gasstation" as the controller route is already set